public/doc/openapi-3-v1.0.0.json

```diff
@@ -806,6 +806,36 @@
         }
       }
     },
+    "/graphql": {
+      "post": {
+        "summary": "Perform a GraphQL Query",
+        "operationId": "postGraphQL",
+        "description": "Performs a GraphQL Query",
+        "requestBody": {
+          "content": {
```

Comment:
@abellotti As we talked about, perhaps this section should include the format of the request as indicated by https://graphql.org/learn/serving-over-http/#post-request. I think you said this piece was auto-generated by the common gem, so it would likely have to change there instead of here.

Reply:
Thanks @eclarizio, I'll create follow-up PRs re: query/operationName/variables, as well as the data/errors in the response.

Also, according to  https://graphql.org/learn/serving-over-http/#post-request we could add support for the GET equivalent ie. GET /api/topological-inventory/v1.0/graphql?query={ ... }